staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -77,6 +77,14 @@ type PodDisruptionBudgetStatus struct {
 
 	// total number of pods counted by this disruption budget
 	ExpectedPods int32 `json:"expectedPods" protobuf:"varint,6,opt,name=expectedPods"`
+
+	// Conditions contain conditions for PDB
```

Comment:
since there are well-known condition types/reasons used by the controller, describe them here

Reply:
Added.